tests/fail-dep/libc/libc-epoll-data-race.rs

```diff
@@ -3,6 +3,7 @@
 //! and therefore still report a data race for things that need to see the second event
 //! to be considered synchronized.
 //@only-target: linux
+//@only-target: android
```

Comment:
This means it will only run on targets that are both linux and android. Which is no target.

You need to use this instead:
```suggestion
//@only-target: linux android
```